addon/ng2/blueprints/ng2/files/karma.conf.js

```diff
@@ -2,10 +2,16 @@ module.exports = function(config) {
   config.set({
     basePath: '',
     frameworks: ['jasmine'],
+    plugins: [
+      require('karma-jasmine'),
+      require('karma-chrome-launcher')
```

Comment:
it works for me if I do `$(npm bin)/karma start` but not if I run from global bin `karma start`. I made sure that I have the same version of karma in both. I think this is because karma loads sibling node modules that start with `karma-` automatically, which is the case for local modules, but not for global modules. If it works globally for you it might be because you installed those plugins globally at some point.